client/app/filters/index.js

```diff
@@ -57,22 +56,7 @@ export function scheduleHumanize(schedule) {
     return 'Never';
   }
   const { count, interval } = secondsToInterval(schedule.interval);
-  let scheduleString = `Every ${count} ${interval} `;
-
-  if (schedule.time) {
-    const parts = schedule.time.split(':');
-    const localTime = moment.utc()
-      .hour(parts[0])
-      .minute(parts[1])
-      .local()
-      .format('HH:mm');
-    scheduleString += `at ${localTime} `;
-  }
-
-  if (schedule.day_of_week) {
-    scheduleString += `on ${schedule.day_of_week}`;
-  }
-  return scheduleString;
+  return `Every ${count} ${interval} `;
```

Comment:
What will it print for "Every Tuesday at 16:00"?

Reply:
"Every 1 week(s)"

Reply:
Sure I can make it into a new component 👍🏻

Reply:
The benefit of implementing the component is that users with read only access to the query will be able to see the real schedule (otherwise it's available only in the dialog).

Reply:
Done!